lib/ansible/plugins/callback/hipchat.py

```diff
@@ -80,11 +90,17 @@ def __init__(self):
             self._display.warning('The `prettytable` python module is not installed. '
                                   'Disabling the HipChat callback plugin.')
 
-        self.msg_uri = 'https://api.hipchat.com/v1/rooms/message'
         self.token = os.getenv('HIPCHAT_TOKEN')
         self.room = os.getenv('HIPCHAT_ROOM', 'ansible')
         self.from_name = os.getenv('HIPCHAT_FROM', 'ansible')
         self.allow_notify = (os.getenv('HIPCHAT_NOTIFY') != 'false')
+        self.hipchat_api_version = os.getenv('HIPCHAT_API_VERSION', 'v1')
```

Comment:
You should not need this anymore, you should have `self.get_option('api_version')` starting in 2.4 (rest of plugin needs updating)

Reply:
also the new plugin config framework allows for ini entries in ansible.cfg